Week3/Homework/Project/Credit-Card-Validator.js

```diff
@@ -0,0 +1,90 @@
+function cardChecker(cardNumber) {
+
+    if (cardNumber * 10 % 2 !== 0) {        // Check if cardNumber is not a number (if it has letters or symbols)
+
+        const newCardNumber = prompt("Invalid Characters, Please Enter Just Numbers :");
+        return cardChecker(newCardNumber);
+
+    }
+
+    else if (cardNumber * 10 % 2 === 0) {       // Check if cardnumber is a number
+
+        const copyOfCardNumber = cardNumber;
+
+        if (copyOfCardNumber.length !== 16) {
+
+            const newCardNumber = prompt("The card number that you entered has not 16 digits, Please Enter a Valid Card Number :");
+            return cardChecker(newCardNumber);
+
+        }
+
+        else if (copyOfCardNumber.length === 16) {      // Check if copyOfCardNumber has 16 digits
```

Comment:
Again, you don't really need the `else-if` block here as a simple `else` block would suffice.